plugins/inputs/t128_peer_path/t128_peer_path.go

```diff
@@ -34,13 +34,15 @@ var peerPathFields = map[string]string{"status": "status", "enabled": "enabled"}
 var peerPathTags = map[string]string{
 	"node":             "node",
 	"adjacentAddress":  "adjacentAddress",
-	"adjacentHostname": "adjacentHostname",
 	"deviceInterface":  "deviceInterface",
 	"networkInterface": "networkInterface",
 	"vlan":             "vlan",
 	"routerName":       "allPeers/nodes/routerName",
 }
-var peerPathQuery = "query {\nallPeers{\nnodes{\npaths{\nadjacentAddress\nadjacentHostname\ndeviceInterface\nenabled\nnetworkInterface\nnode\nstatus\nvlan}\nrouterName}}}"
+
+var peerPathHostnameQuery = "query {\nallPeers{\nnodes{\npaths{\nadjacentAddress\nadjacentHostname\ndeviceInterface\nenabled\nnetworkInterface\nnode\nstatus\nvlan}\nrouterName}}}"
```

Comment:
It could be nice for these to be written out for readability:
```go
const peerPathHostnameQuery = `query {
    allPeers {
       nodes {
            paths {
                ...
            }
        }
    }
}
```